### PR TITLE
Code Review #1: Newsweek clone with Bootstrap

### PR DESCRIPTION
**Pull request by wickathou**
This is the submission for code review. The page features responsive design and extensive use of the Bootstrap framework.

### CODE REVIEW

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @wickathou 

**Excellent work on this project, however, you have to fix the following before we can move on ⬇️** 

1. Fix the CSS linter errors on the project, you can find out more on about setting up stickler on [here](https://github.com/microverseinc/linters-config/tree/master/css).

2. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input).


### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by wickathou** [comment]
- Updated CSS, all errors removed
- Updated HTML, removed all errors

File: index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="zxx">
```
Comment (Redvanisation) [inline index.html:2]: The `<lang>` attribute should refer to the language used on the page for better SEO.

**Review by Redvanisation — APPROVED** [review]
### _Nicely done, it seems that you have met the requirements and I am approving your project. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

_I have also made a comment below, please consider it in the future._ ⬇️ 

### Good luck! 🍀

**Pull request merged**